### DISCUSSION TITLE
Centos7已经配置了clash科学上网，如何使rsshub调用clash的proxy？

### DISCUSSION THREAD

**Discussion by FutureDecade** [DIYgod/RSSHub#12014] [Q&A] [answered]
### 这是一个什么样的功能？

![屏幕截图 2022-07-12 075213](https://user-images.githubusercontent.com/14036193/178377613-67e5385a-6b1f-4fdc-8903-5d7047efa1d5.png)
国内腾讯云服务器，使用docker安装了rsshub+ttrss之后，发现rsshub的很多订阅源无法抓取，所以配置了Clash+飞机来实现科学上网，curl google.com可以通过。但是，rsshub仍然无法抓取。搜索相关问题发现，在rsshub里还需要通过配置来调用clash的proxy。但这一步该怎么做呢？请教一下。

### 这个功能可以解决什么问题？

解决国内服务器rsshub订阅源的抓取

### 这不是重复的功能请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保这项功能尚未被请求。

**Comment by nczitzk** [comment, marked as answer]
请查看 [文档](https://docs.rsshub.app) 中 [代理配置](https://docs.rsshub.app/install/#pei-zhi-dai-li-pei-zhi) 部分的描述。
